task/template/src/main.rs

```diff
@@ -18,3 +14,5 @@ fn main() -> ! {
         // will turn this into a single undefined instruction.
     }
 }
+
+include!(concat!(env!("OUT_DIR"), "/config.rs"));
```

Comment:
Similar to build.rs, there should be a comment here indicating that this line should be dropped if build.rs doesn't generate config.rs.